application/src/main/resources/application.properties

```
@@ -1,7 +1,7 @@
 spring.application.name=application
-spring.datasource.url=${database.url}
-spring.datasource.username=${database.username}
-spring.datasource.password=${database.password}
+spring.datasource.url=jdbc:mysql://localhost:3306/dev_movie
+spring.datasource.username=root
+spring.datasource.password=1111
```

민감 정보는 숨겨주세요.

`resources/privates/private.properties` 에 아래처럼 적으시면 됩니다.

```
database.url=jdbc:mysql://localhost:3306/dev_movie
database.username=root
database.password=1111
```
```suggestion
spring.datasource.url=${database.url}
spring.datasource.username=${database.username}
spring.datasource.password=${database.password}
```

---

감사합니닷